Pipfile

```diff
@@ -5,49 +5,50 @@ name = "pypi"
 
 [packages]
 # JupyterHub and system libraries (e.g., Spark, MinIO, etc.)
-jupyterlab= "==4.2.5"
+jupyterlab= "==4.3.5"
 jupyterhub = "==5.2.1"
 jupyterhub-nativeauthenticator = "==1.3.0"
 dockerspawner = "==13.0.0"
 jupyterlab_favorites = "==3.2.2"
 sidecar = "==0.7.0"
-json5 = "==0.9.25"
-pycurl = "==7.45.3"
-pyspark= "==3.5.3"
-boto3 = "==1.35.48"
-minio = "==7.2.10"
+json5 = "==0.10.0"
+pycurl = "==7.45.4"
+pyspark= "==3.5.4"
+boto3 = "==1.36.11"
+minio = "==7.2.15"
 delta-spark = "==3.2.0"  # should match JAR version (DELTA_SPARK_VER) specified in the Dockerfile
 PyMySQL = "==1.1.1"
-urllib3 = "==2.2.3"
-jupyterlab-git = "==0.50.1"
+urllib3 = "==2.3.0"
+jupyterlab-git = "==0.50.2"
 cdm-jupyterlab-brand-extension = {file = "https://github.com/kbaseincubator/cdm_jupyterlab_brand_extension/raw/main/dist/cdm_jupyterlab_brand_extension-0.1.0-py3-none-any.whl"}
+cacheout = "==0.16.0"
```

Comment:
This seems to be a new install, not an update?

Reply:
yup yup. Will be used in the next PR.